service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/azure/storageContainer/VerifyAzureStorageContainerCanBeCreatedStep.java

```diff
@@ -137,6 +135,12 @@ private StepResult validateWorkspaceStorageAccountExists(
     return StepResult.getStepResultSuccess();
   }
 
+  private static void putStorageAccountNameAndRegionInWorkingMap(
+      FlightContext context, String storageAccount, String storageAccount1) {
```

Comment:
`storageAccount1` seems inaccurate for the parameter name -- `storageAccountRegion` perhaps?

Reply:
oops nice catch